docs/widgetbook-cloud/hosting.mdx

```diff
@@ -0,0 +1,224 @@
+# Widgetbook Hosting
+
+The Widgetbook Hosting allows users to upload the Flutter web version of a Widgetbook to Widgetbook Cloud.
+To upload any widgetbook, you require an API key.
+
+## How to create an API key
+
+To create an API key, you need to login with your account at [app.widgetbook.io](https://app.widgetbook.io). 
+API keys are bound to projects. 
+Therefore, you require a project to retrieve an API key. 
+Furthermore, only users with the `Owner` role can fetch the API key of a project. 
```

Comment:
Explanation why only users can fetch the API key. 

Plus something like "If you are not the owner of your project but would still like to fetch the API key, please ask your owner to promote you from member to owner. For information on member management, please see [member management]"